src/platform/forms/components/review/SubmitPending.jsx

```diff
@@ -0,0 +1,48 @@
+// libs
+import React from 'react';
+
+// platform - forms components
+import Column from 'platform/forms/components/common/grid/Column';
+import Row from 'platform/forms/components/common/grid/Row';
+
+// platform - forms containers
+import PreSubmitSection from 'platform/forms/containers/review/PreSubmitSection';
+
+// platform - forms-system components
+import ProgressButton from 'platform/forms-system/src/js/components/ProgressButton';
+
+function SubmitPending(props) {
+  const { formConfig, goBack, onSubmit } = props;
+
+  return (
+    <>
+      <Row>
+        <Column className="columns" />
+      </Row>

```

Comment:
Why?

Reply:
Just keeping with the spirit of the previous code. Same as above where `submitMessage` was never set and therefore undefined.

Reply:
cleaned up className